### PR TITLE
{vis}[GCCcore/13.2.0] gnuplot v6.0.1, libcerf v2.4

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by furstj** [comment]
Test report by @furstj
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
think - Linux openSUSE Leap 15.6, x86_64, Intel(R) Core(TM) i7-10700T CPU @ 2.00GHz, 1 x NVIDIA Quadro P1000, 550.90.07, Python 3.11.9
See https://gist.github.com/furstj/f68ee5547b71800dbebf2d000de39bd6 for a full test report.

**Comment by furstj** [comment]
Test report by @furstj
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in total)
think - Linux openSUSE Leap 15.6, x86_64, Intel(R) Core(TM) i7-10700T CPU @ 2.00GHz, 1 x NVIDIA Quadro P1000, 550.90.07, Python 3.11.9
See https://gist.github.com/furstj/c90fb0f40b9c0a90742983fdc8867395 for a full test report.

**Review by boegel — DISMISSED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3107.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/2d53dcc5354df9dec5ddbb03cccef5d5 for a full test report.

**Comment by boegel** [comment]
Hmm, this suffers from the same problem as #21330:
```
undefined reference to `libiconv_open'
```

**Comment by akesandgren** [comment]
The one for 13.3.0 does
```
# fix undefined reference to symbol 'libiconv_open'
preconfigopts += ' export LDFLAGS="-Wl,--copy-dt-needed-entries" && '
```

File: easybuild/easyconfigs/g/gnuplot/gnuplot-6.0.1-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+# make sure that right Lua library is used (bypassing pkg-config)
+preconfigopts += 'export LUA_CFLAGS="-I$EBROOTLUA/include" && export LUA_LIBS="$EBROOTLUA/lib/liblua.a" && '
+
+# fix undefined reference to symbol 'libiconv_open'
+preconfigopts += ' export LDFLAGS="-Wl,--copy-dt-needed-entries" && '
+
+configopts = '--with-qt=qt6 --without-latex --disable-wxwidgets'
+
+buildopts = 'LDFLAGS="$LDFLAGS -Wl,--copy-dt-needed-entries" '
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/gnuplot/gnuplot-6.0.1-GCCcore-13.2.0.eb:44]: I don't think you need this one, only the preconfigopts one is used in the version for 13.3.0

File: easybuild/easyconfigs/g/gnuplot/gnuplot-6.0.1-GCCcore-13.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+buildopts += 'LIBS="-lcerf -lcairo -lpangocairo-1.0"'
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/gnuplot/gnuplot-6.0.1-GCCcore-13.2.0.eb:45]: Do you really need this one? cmp 13.3.0 version

File: easybuild/easyconfigs/g/gnuplot/gnuplot-6.0.1-GCCcore-13.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+preconfigopts = 'autoreconf && '
+
+# make sure that right Lua library is used (bypassing pkg-config)
+preconfigopts += 'export LUA_CFLAGS="-I$EBROOTLUA/include" && export LUA_LIBS="$EBROOTLUA/lib/liblua.a" && '
+
+# fix undefined reference to symbol 'libiconv_open'
+preconfigopts += 'export LDFLAGS="-Wl,--copy-dt-needed-entries" && '
+
+configopts = '--with-qt=qt6 --without-latex --disable-wxwidgets'
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/gnuplot/gnuplot-6.0.1-GCCcore-13.2.0.eb:42]: The with-qt6 is not strictly needed, but doesn't hurt either.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/akesandgren/11f5e2a6f12cb4fa72fe14f640bf199b for a full test report.

**Comment by akesandgren** [comment]
@boegelbot Please test @ jsc-zen3

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**